spec/instant.html

```diff
@@ -647,5 +625,30 @@ <h1>TemporalInstantToString ( _instant_, _timeZone_, _precision_ )</h1>
         1. Return the string-concatenation of _dateTimeString_ and _timeZoneString_.
       </emu-alg>
     </emu-clause>
+    <emu-clause id="sec-temporal-differencetemporalinstant" aoid="DifferenceTemporalInstant">
+      <h1>
+        DifferenceTemporalInstant (
+          _first_: a Temporal.Instant,
+          _second_: a Temporal.Instant,
+          _options_: an Object or *undefined*,
```

Comment:
```suggestion
          _options_: an ECMAScript language value,
```

Reply:
Why should we annotate _options_ as an ECMAScript language value for some type but not the other types?

Reply:
I made the comments based on whether the _options_ received from the JS script had already been passed through GetOptionsObject. (All JS script values are "an ECMAScript language value")